packages/node_modules/@ciscospark/internal-plugin-user/src/user.js

```diff
@@ -129,7 +129,12 @@ const User = SparkPlugin.extend({
       resource: `users`
     })
       .then((res) => res.body)
-      .then(tap((user) => this.recordUUID(user)));
+      .then(tap((user) => this.recordUUID({
+        id: user.id,
+        // CI endpoints don't use the same user format as actors, so, email may
+        // be in one of a few fields
+        emailAddress: user.email || user.emailAddress
```

Comment:
we're having to massing conversation/user fields in the widget too...

Reply:
?

Reply:
wow, total keyboard fail on that. I don't even know what I was trying to say other than we are doing something similar on widgets.